demo/full/scripts/controllers/ContentList.jsx

```diff
@@ -27,7 +27,7 @@ const CONTENTS_PER_TYPE = TRANSPORT_TYPES.reduce((acc, tech) => {
       if (IS_HTTPS) {
         if (content.url.startsWith("http:")) {
           name = "[HTTP only] " + name;
-          disabled = true;
+          disabled = false;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Review this useless assignment: "disabled" already holds the assigned value along all execution paths. ([javascript:S4165](https://sonarcloud.io/organizations/rx-player/rules#rule_key=javascript%3AS4165))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=rx-player&pullRequest=366&issues=AWbt7Z40W3Y69bz5aGXP&open=AWbt7Z40W3Y69bz5aGXP)

[ISSUE_KEY]: AWbt7Z40W3Y69bz5aGXP